docs/security/03-07-production-profile.md

```diff
@@ -0,0 +1,85 @@
+---
+title: OAuth2 server production profile
+type: Details
+---
+
+By default, every Kyma deployment is installed with the OAuth2 server using what is considered a development profile. In the case of the ORY Hydra OAuth2 server, this means that:
+  - Hydra works in the "in-memory mode" and saves the registered OAuth2 clients in its memory, without the use of persistent storage.
+  - Similarly to other Kyma components, it has an Istio sidecar Pod injected.
+
+This configuration is not considered production-ready. To use the Kyma OAuth2 server in a production environment, configure Hydra to use the production profile.
+
+## The production profile
+
+The production profile introduces the following changes to the Hydra OAuth2 server deployment:
+   - Persistence is enabled for Hydra; the registered client data is saved in an in-cluster database.
+   - The Hydra OAuth2 server and the Hydra Maester controller have Istio sidecars disabled, destinationRule custom resources are created for these components.
```

Comment:
Istio sidecars should be independent of the profile, meaning disabled by default. There's no need to add sidecar on dev profile if we do not have in production :)

Reply:
UDPATE : oh crap, without synchro mode we cannot disable the sidecars globally because it will take effect after restarting pod === losing data. Until synchronization mode will be in place then we need to leave it this way.